external/vcm/vcm/data_transform.py

```diff
@@ -250,6 +251,37 @@ def implied_downward_radiative_flux_at_surface(
     return ds
 
 
+@register(
+    [
+        "Qm",
+        DELP,
+        DLW_SFC,
+        DSW_SFC,
+        DSW_TOA,
+        ULW_SFC,
+        ULW_TOA,
+        USW_SFC,
+        USW_TOA,
+        LHF,
+        SHF,
+    ],
+    ["implied_downward_radiative_flux_at_surface"],
+)
+def implied_downward_radiative_flux_at_surface_no_fine_res_t_nudging(ds, rectify=True):
```

Comment:
I realize there are no type hints below, but maybe nice to add in new code.

Reply:
I reverted the commit that added this code.